src/everest_models/jobs/fm_well_trajectory/well_trajectory_resinsight.py

```diff
@@ -22,7 +22,8 @@ class ResInsight:
     def __init__(self, executable: Optional[Path] = None) -> None:
         if executable is None:
             executable = shutil.which("ResInsight")
-        self._executable = executable
+        # ResInsight only checks for empty string and not None
+        self._executable = executable if executable is not None else ""
```

Comment:
I am changing this in PR #21 which also changes the way we test. This change would not be necessary anymore since that PR sets the executable default to `""`.